### ISSUE TITLE
UI: (13.C) Inventory Save Search has Insufficient Contrast

### ISSUE THREAD

**Issue by hawkishpolicy** [cisagov/XFD#342]
# 🐛 Summary #

Insufficient contrast is provided for “save search” button on the Inventory> Search Results page. The color difference of text and its background should have a contrast ratio of at least 4.5:1. Using contrast analyzers, spot test various text and backgrounds throughout the site or application to ensure a contrast ratio of at least 4.5:1. When insufficient contrast issues are found, adjust either one or the other color to meet a minimum of 4.5:1 ratio.

## To reproduce ##

1. Use contrast checker against "save search" button and background in the Search Results section of Inventory.

## Expected behavior ##

What did you expect to happen that didn't?

Should have contrast ratio of at least 4.5:1.

## Any helpful log output or screenshots ##

Add any screenshots of the problem here.
![image](https://github.com/cisagov/XFD/assets/123108455/8e1a8210-2155-4d3a-a885-d744915cea09)
